email.go

```diff
@@ -0,0 +1,49 @@
+package customerio
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"io"
+)
+
+type Email struct {
+	TransactionalMessageID int                    `json:"transactional_message_id,omitempty"`
+	CustomerID             string                 `json:"customer_id"`
+	To                     string                 `json:"to,omitempty"`
+	From                   string                 `json:"from,omitempty"`
+	Subject                string                 `json:"subject,omitempty"`
+	Body                   string                 `json:"body,omitempty"`
+	MessageData            map[string]interface{} `json:"message_data,omitempty"`
+	BCC                    string                 `json:"bcc,omitempty"`
+	ReplyTo                string                 `json:"reply_to,omitempty"`
+	FromID                 int                    `json:"from_id,omitempty"`
+	ReplyToID              int                    `json:"reply_to_id,omitempty"`
+	Headers                map[string]string      `json:"headers,omitempty"`
+	Attachments            map[string]string      `json:"attachments,omitempty"`
+	FakeBCC                *bool                  `json:"fake_bcc"`
+	HideBody               *bool                  `json:"hide_body"`
```

Comment:
I thought we'd decided on something other than HideBody that reflected more that we're not going to store the body.